api/envoy/extensions/filters/http/bandwidth_limit/v3alpha/bandwidth_limit.proto

```diff
@@ -67,4 +67,17 @@ message BandwidthLimit {
   // Runtime flag that controls whether the filter is enabled or not. If not specified, defaults
   // to enabled.
   config.core.v3.RuntimeFeatureFlag runtime_enabled = 5;
+
+  // Optional flag to enabled response trailers.
+  // Default is false.
```

Comment:
Default and optional are redundant for scalars, so just say `Enable response trailers` and link to documentation or explain what they are.

Reply:
Also, single or plural response trailers?

Reply:
updated, try to make it more clear.